### PR TITLE
pythonPackages.pytest-check: 0.3.9 -> 1.0.1, staticjinja: 0.4.0 -> 1.0.3

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pytest-check/default.nix [hunk 1]
```diff
@@ -7,15 +7,14 @@
...
+  version = "1.0.1";
 
   src = fetchPypi {
     pname = "pytest_check";
     inherit version;
-    sha256 = "0asrrz0fgk6wqffsz1ffd6z9xyw314fwh5bwjzcq75w8w1g4ass9";
+    sha256 = "1i01i5ab06ic11na13gcacrlcs2ab6rmaii0yz0x06z5ynnljn6s";
   };
 
-  propagatedBuildInputs = [ pytest ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pytest-check/default.nix:18]: Please remove pytest from inputs.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pytest-check/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/python-modules/pytest-check/default.nix:18]: But `pytest` should be in `propagatedBuildInputs`: https://github.com/okken/pytest-check/blob/1.0.0/pyproject.toml#L11

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/pytest-check/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/development/python-modules/pytest-check/default.nix:18]: Oops, yeah. Fixed.

**Comment by fgaz** [comment]
Ugghh.... I used the wrong ref in the rebase command... let me fix that...

**Comment by fgaz** [comment]
Fixed

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110776` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python37Packages.pytest-check</li>
<li>python37Packages.staticjinja</li>
<li>python38Packages.pytest-check</li>
<li>staticjinja (python38Packages.staticjinja)</li>
<li>python39Packages.pytest-check</li>
<li>python39Packages.staticjinja</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110776` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python37Packages.pytest-check</li>
<li>python37Packages.staticjinja</li>
<li>python38Packages.pytest-check</li>
<li>staticjinja (python38Packages.staticjinja)</li>
<li>python39Packages.pytest-check</li>
<li>python39Packages.staticjinja</li>
</ul>
</details>

**Pull request merged**